templates/todo/projects/java-mongo-aca/.repo/bicep/infra/main.bicep

```diff
@@ -25,19 +25,15 @@ param logAnalyticsName string = ''
 param resourceGroupName string = ''
 param webContainerAppName string = ''
 param apimServiceName string = ''
+param apiAppExists bool = false
+param webAppExists bool = false
```

Comment:
`main` module needs a signal to detect whether or not the container image should be set or to use existing image.